modules/common/go.mod

```diff
@@ -79,3 +79,7 @@ require (
 	github.com/kr/pretty v0.3.1 // indirect
 	github.com/rogpeppe/go-internal v1.10.0 // indirect
 )
+
+// mschuppert: map to latest commit from release-4.13 tag
+// must consistent within modules and service operators
+replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230414143018-3367bc7e6ac7
```

Comment:
Is there a reason to do this as a replace instead of updating L11 directly?

Reply:
It might be because various `go` commands automatically find `github.com/openshift/api v3.9.0+incompatible` and overwrite what you may have manually added to the `require` section.  At least I feel I remember seeing that happen to me in the past.

Reply:
we could for common module, but where its indirect you'd have to use replace. I thoughts its more obvious then to do it the same everywhere.

Reply:
Thanks. It seems the indirect dep in modules/test coming from some service operator /api module. so we cannot avoid that.